modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -55,39 +56,47 @@ When specifying a name, be sure to use the accepted syntax expression `^[a-zA-Z0
 
 <8> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For an Intel network interface controller (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
 
-<9> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
+<9> Set `externallyManage` to `true` to ensure some virtual functions (VFs) are reserved for host managed systems. With the value set to `false` the SR-IOV Network Operator manages and configures all allocated VFs.
```

Comment:
maybe something like 

```
Allow the SR-IOV network operator to configure and allocate a subset of virtual functions from
a physical function that is configured externally from SR-IOV network operator.
```

please take a look on the design proposal in the sriov repo https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/476/files